doc/adr/0009-make-offline-mode-file.md

```diff
@@ -19,26 +19,67 @@ SQLite plugin for Flutter. Supports iOS, Android and MacOS.
 
 Usage 
 
-- Opening a database
-```
-Future<Database> openDatabase(String path, {OpenDatabaseOptions options});
-```
-- Insert record
+- Table Document: Help save documents when it make available offline in MySpace
+
 ````
-Future<int> insert(String table, Map<String, dynamic> values)
+Table Document {
```

Comment:
what objects to describe the structure in your code?

Reply:
WDYM? I have described the table structure for the document file here with its fields as clear.